go.mod

```diff
@@ -110,3 +110,5 @@ require (
 	sigs.k8s.io/structured-merge-diff/v4 v4.2.3 // indirect
 	sigs.k8s.io/yaml v1.3.0 // indirect
 )
+
+replace github.com/konveyor/tackle2-addon => /Users/shurley/repos/tackle2-addon
```

Comment:
Reminder to remove this before merged. ^^

Reply:
updated now that the add-on PR is merged!